javascript/javascript2/week3/fastestpresser.js

```diff
@@ -0,0 +1,65 @@
+let gameForm = document.getElementById("game-start-form");
+let timeDuration = document.getElementById("duration");
+let sCountNumber = document.getElementById("scount");
+let lCountNumber = document.getElementById("lcount");
+let sCountWin = document.getElementById("scountWinner");
+let lCountWin = document.getElementById("lcountWinner");
+let restartButton = document.getElementById("restart");
+let sCount = 0;
+let lCount = 0;
+document.addEventListener("keydown", keyCount);
+function keyCount(event)
+ {
+    if (timeDuration.value > 0) {
+    if(event.code === "KeyL"){
+        lCount++;
+        lCountNumber.innerHTML = lCount;
+        lCountWin.innerHTML = ""; 
+        sCountWin.innerHTML = "";
+    }
+    if(event.code === "KeyS"){
+        sCount++;
+        sCountNumber.innerHTML = sCount;
+        lCountWin.innerHTML = ""; 
+        sCountWin.innerHTML = "";
+    }
+}
+}
+gameForm.addEventListener("click",(event)=>{
+    if (timeDuration.value > 0) {    
```

Comment:
To reduce code indentation, you can return early.
```suggestion
    if (timeDuration.value <= 0) {
        return;
    }
```

Reply:
Changes done.